index.html

```diff
@@ -92,41 +89,41 @@ <h4>ONE ON ONE COACHING</h4>
         </div>
 
 </section>
-<section class="row">
+<footer class="row">
         <div class="contact col-12 md-col-6">
             <div class="col-3">
             
             </div>
             <div class="contacttext col-7">
-                
-                
-                    <h3>CONTACT</h3>
-                    <p>500 Terry Francois Street<br>San Francisco, CA 94158<br><br>
-                    Tel: 123-456-7890<br>Fax:123-456-7890<br><br>info@mysite.com</p>
-                    <i class="fa fa-facebook"></i>
-                    <i class="fa fa-twitter"></i>
-                    <i class="fa fa-youtube"></i>
-                    <i class="fa fa-instagram"></i>   
+                    <div><h3>CONTACT</h3></div>
+                    <div><p>500 Terry Francois Street<br>San Francisco, CA 94158<br><br>
+                    Tel: 123-456-7890<br>Fax:123-456-7890<br><br>info@mysite.com</p></div>
+                    <div>
+                        <i class="fa fa-facebook"></i>
+                        <i class="fa fa-twitter"></i>
+                        <i class="fa fa-youtube"></i>
+                        <i class="fa fa-instagram"></i>
+                    </div>   
             </div>
             <div class="col-2">
                 
             </div>
         </div>
-
+    
      <div class="form col-12 md-col-6">
-        <p>Enter Your Name</p>
-        <input type="text"><br>
-        <p>Enter Your Email</p>
-        <input type="text"><br>
-        <p>Enter Your Subject</p>
-        <input type="text"><br>
-        <p>Message</p>
-        <input type="text" id="message"><br>
-        <input type="submit" id="submit"><br>
+        <div><label>Enter Your Name:</label></div>
```

Comment:
You can use the `for` label to link your text to the relevant `input` element, e.g.:

```html
<label for="subject">Your Subject:</label>
<input id="subject" type="text" />
```

If you do this, when the user clicks on the description ("Your Text"), the browser will automatically place the cursor in the input element and focus it. If they're on a mobile device, this will also automatically show the keyboard.

![](https://media.giphy.com/media/lNQvuXkUnQqtIjPadD/source.gif)